.github/workflows/nlu-hyperopt.yml

```diff
@@ -0,0 +1,17 @@
+
+name: main
+on: [pull_request]
+
+jobs:
+  nlu-hyperopt:
+    name: NLU hyperoptimization
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: RasaHQ/nlu-hyperopt@rasa-1-compat
+      name: Run NLU Hyperoptimization
+      env:
+        DATA_DIR: $GITHUB_WORKSPACE/data
+        SEARCH_SPACE: $GITHUB_WORKSPACE/nlu_hyperopt/space.py
+        MAX_EVALS: 1
```

Comment:
probably be a weird example value for an example since it only tries out 1 configuraration. Either increase it or add a comment.